### DISCUSSION TITLE
Generate manifest for store apps

### DISCUSSION THREAD

**Discussion by leocoolmitra** [microsoft/winget-cli#2160] [General]
When we use wingetcreate new and then put the url of the installer the manifest generator comes up and helps us to create the manifest and then we can store it on a file share and user can use winget install -m "the path of the manifest file".But is there a way by which we can generate manifest for store apps so that we can share it on a file share so other users can use it using winget install -m

**Comment by denelon** [comment]
This isn't a supported scenario. I'd like to understand why using the "msstore" source doesn't meet your needs.

**Reply by leocoolmitra** [reply]
I can use the ms store source but what if i want to deploy this package to all my users and they can install it using winget and i can host it in a private source. Just like private store the way we have in microsoft business store because many enterprises might want to use winget to let their employees install apps and restrict it to a particular source.

**Reply by leocoolmitra** [reply]
so that the store remains disabled in the environment and also the users can install only the apps which are provided. Like in microsoft business store the admins provide the only apps that they want their users to download.Like the same way if we can store the manifest files on a file share and the end user can install it using winget install -m "the path of the manifest file" and force the end users to install only apps from that source which would give the system administrators a complete control what their end users can install
